UnityGLTF/Assets/UnityGLTF/Scripts/Cache/AssetCache.cs

```diff
@@ -27,13 +33,18 @@ public class AssetCache : IDisposable
 		/// <summary>
 		/// Byte buffers that represent the binary contents that get parsed
 		/// </summary>
-		public Dictionary<int, BufferCacheData> BufferCache { get; private set; }
+		public BufferCacheData[] BufferCache { get; private set; }
```

Comment:
I use arrays where possible since we know that we have fixed sized. I have to use a List in one case because it would be a 2D array and we don't precalculate the second dimension.